.appveyor.yml

```diff
@@ -0,0 +1,83 @@
+version: 1.0.{build}
+
+# MySQL fails to link on 2015
+image: Visual Studio 2013
+
+branches:
+  only:
+    - master
+    - appveyor_nonsense
```

Comment:
Yes, I keep a separate branch that I can just push to for when I need to debug CI things. On travis it's called `ಠ_ಠ` but Windows doesn't like unicode.